packages/bpk-scrim-utils/src/withScrimmedPortal-test.tsx

```diff
@@ -105,4 +106,48 @@ describe('withScrimmedPortal', () => {
 
     expect(screen.getByText('Wrapped Component / portal is not ready yet / portal is now ready')).toBeInTheDocument();
   });
+
+  it('renders the wrapped component inside a portal correctly when runOnServer is false (default)', () => {
+    const DialogContent = () => <div>Dialog content</div>;
+    const ScrimmedComponent = withScrimmedPortal(DialogContent);
+
+    render(
+      <div id="pagewrap">
+        <div> Content hidden from AT</div>
+        <ScrimmedComponent
+          getApplicationElement={() => document.getElementById('pagewrap')}
+        />
+      </div>
+    );
+    expect(document.body).toMatchSnapshot();
```

Comment:
can we use more specific expects? snapshots tend to offer a poor testing experience in the long term https://medium.com/@sapegin/whats-wrong-with-snapshot-tests-37fbe20dfe8e

Reply:
Hmm, I agree sometimes the snapshot tests are pretty useless, but in this case it really is exactly the tree that we want to check, but I guess we can change it to something like - element X is a descendant of pagewrap, element Y is not.

Reply:
Agree with Arnau here, snapshot testing is against our advice, see [our documentation](https://web-documentation.skyscannertools.net/testing/unit-testing-granular-details.html#snapshot-testing):

> TLDR: Avoid using snapshots for testing components where possible
>
> Good tests encode the developers intent, but snapshot tests lack this. Inevitably any change to the component breaks the snapshot, so engineers become habituated to blindly accepting changes to snapshots without verifying whether the change is desired or not. This is particularly bad for large snapshots.

Reply:
I've removed the snapshot tests and added new tests